src/content_mapblock.cpp

```diff
@@ -1316,19 +1315,45 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 			p2.X++;
 			MapNode n2 = data->m_vmanip.getNodeNoEx(blockpos_nodes + p2);
 			const ContentFeatures *f2 = &nodedef->get(n2);
-			if(f2->drawtype == NDT_FENCELIKE)
+			if(f2->drawtype == NDT_FENCELIKE || f2->drawtype == NDT_NORMAL)
```

Comment:
On December 16, 2015 9:25:04 AM PST, BlockMen <notifications@github.com> wrote:
>> @@ -1316,19 +1315,45 @@ void
>mapblock_mesh_generate_special(MeshMakeData *data,
>>  			p2.X++;
>>  			MapNode n2 = data->m_vmanip.getNodeNoEx(blockpos_nodes + p2);
>>  			const ContentFeatures *f2 = &nodedef->get(n2);
>> -			if(f2->drawtype == NDT_FENCELIKE)
>> +			if(f2->drawtype == NDT_FENCELIKE || f2->drawtype == NDT_NORMAL)
>
>Please check for a group value, like ````connect_to_fence``` instead of
>hardcoding it here.
>
>EDIT: see how raillike does it, for example
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/minetest/minetest/pull/3459/files#r47805805

As said in the PR, that's planned to be added in a subsequent patch. I wanted to provide a working set of changes that people can test and see first, before doing the api level work.